### PR TITLE
Migrating VsTestPlatformInstaller Task to Node10

### PR DESCRIPTION
**Pull request by sudha7562**
**Task name**: Migrate VsTestPlatformToolInstallerV1 to NodeJS10
**Description**: Task Upgraded to NodeJS 10 

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Testing:** 
 - Build succeeded on local
image

 - Passed all the unit tests
 
image

 - Ran pipeline using the task.
 1. Visual Studio Test Platform Installer task
image
 2. Raw log for Visual Studio Test Platform Installer task
image

**Observation for Pipeline (before and after changes)**
<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:dt="uuid:C2F41010-65B3-11d1-A29F-00AA00C14882"
xmlns="http://www.w3.org/TR/REC-html40">

VsTestPlatformToolInstaller| Node version| VsTestPlatformToolInstaller   (version) | VsTest (Test   platform Version) |  Outcome of the   pipeline
-- | -- | -- | --|--|
1.211.0  | N10 | latest (include pre release) | Installed   by Tool installer | failed
1.211.0  | N10 | latest (include pre release) | Latest | passed
1.211.0  | N10 | Latest Stable | Installed by Tools   Installer | Passed
1.211.0  |  N10 | Latest Stable | Latest |  passed
1.151.3  |  N6 | Latest (Includes Pre-Release) | Installed by Tools Installer |failed
1.151.3  | N6 | Latest (Includes   Pre-Release) | latest | passed
1.151.3 |  N6 | Latest Stable | Installed by Tools   Installer |  Passed
1.151.3  | N6 | Latest Stable | latest | passed

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Comment by triptijain2112** [comment]
@sudha7562 Please update the testing done.

**Comment by triptijain2112** [comment]
@sudha7562 Can you please confirm that the installer specified in VsTestPlatformToolInstallerV1 task was used in Visual studio test task also?

**Review by triptijain2112 — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -1,1710 +1,8 @@
 {
```
Comment (KonstantinTyukalov) [inline package-lock.json:1]: Are we need this changes in global tasks package-lock?

File: Tasks/VsTestPlatformToolInstallerV1/task.loc.json [hunk 2]
```diff
@@ -10,12 +10,12 @@
...
     "DeploymentGroup"
   ],
-  "minimumAgentVersion": "2.103.0",
+  "minimumAgentVersion": "2.144.0",
   "author": "Microsoft Corporation",
   "version": {
     "Major": 1,
-    "Minor": 151,
-    "Patch": 3
+    "Minor": 211,
```
Comment (KonstantinTyukalov) [inline Tasks/VsTestPlatformToolInstallerV1/task.loc.json:17]: Could you bump version according to the current sprint number

File: Tasks/VsTestPlatformToolInstallerV1/task.json [hunk 3]
```diff
@@ -10,12 +10,12 @@
...
         "DeploymentGroup"
     ],
-    "minimumAgentVersion": "2.103.0",
+    "minimumAgentVersion": "2.144.0",
     "author": "Microsoft Corporation",
     "version": {
         "Major": 1,
-        "Minor": 151,
-        "Patch": 3
+        "Minor": 211,
```
Comment (KonstantinTyukalov) [inline Tasks/VsTestPlatformToolInstallerV1/task.json:17]: Could you bump version according to the current sprint number

**Review by KonstantinTyukalov — CHANGES_REQUESTED** [review]

**Review by mpodriezov — APPROVED** [review]

File: package-lock.json (hunk 1 above)
Comment (mpodriezov) [reply package-lock.json:1]: I dont think we need any changes on this level

File: Tasks/VsTestPlatformToolInstallerV1/task.json (hunk 3 above)
Comment (sudha7562) [reply Tasks/VsTestPlatformToolInstallerV1/task.json:17]: Hello,
I have updated the version according to the current sprint, please review it.

Comment (triptijain2112) [reply Tasks/VsTestPlatformToolInstallerV1/task.json:17]: In task.json file, the version still shows as 211, Can you update it to 212?

**Review by mpodriezov — APPROVED** [review]

**Review by triptijain2112 — CHANGES_REQUESTED** [review]
Can we please check use of typings.json file and why have we removed it?

File: Tasks/VsTestPlatformToolInstallerV1/task.json (hunk 3 above)
Comment (sudha7562) [reply Tasks/VsTestPlatformToolInstallerV1/task.json]: Typing get deprecated, using @type is the suggested way to install dependencies, I have followed the doc for upgrading azure pipeline task to node 10.

**Comment by sudha7562** [comment]
> 

Typing get deprecated, using @type is the suggested way to install dependencies, I have followed the doc for upgrading azure pipeline task to node 10.

File: Tasks/VsTestPlatformToolInstallerV1/task.json (hunk 3 above)
Comment (sudha7562) [reply Tasks/VsTestPlatformToolInstallerV1/task.json]: updated the version in task.json file.

**Review by triptijain2112 — APPROVED** [review]

**Review by KonstantinTyukalov — APPROVED** [review]

**Review by rasunkar — COMMENTED** [review]
Looks good

**Pull request merged**